packages/core/src/components/Table/stories/TableHooks/TableHooks.stories.tsx

```diff
@@ -1357,11 +1387,18 @@ const UseHvHeaderGroups = () => {
 
             return (
               <HvTableRow {...row.getRowProps()}>
-                {row.cells.map((cell) => (
-                  <HvTableCell {...cell.getCellProps()}>
-                    {cell.render("Cell")}
-                  </HvTableCell>
-                ))}
+                {row.cells.map((cell) => {
+                  let headers = cell.column.id;
+                  if (cell.column.parent) {
+                    headers += ` ${cell.column.parent.id}_${cell.column.parent.columns.length}`;
+                  }
+
+                  return (
+                    <HvTableCell {...cell.getCellProps()} headers={headers}>
+                      {cell.render("Cell")}
+                    </HvTableCell>
+                  );
+                })}
```

Comment:
I had to do this because, for some reason, react table automatically changes the grouped column id from `gc-event-info` to `gc-event-info_3`. However, when I try to set the `headers` and get the parent `id` with `cell.column.parent.id` the id is `gc-event-info`.

Reply:
However, instead of using `id`, I can use `originalId` for the grouped header. I just have to add this to our TS.

Reply:
This is what happens when we copy the typings - we don't get these fixes/upgrades 😞 
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55634/files